app/models/intent.rb

```diff
@@ -0,0 +1,5 @@
+class Intent < ApplicationRecord
```

Comment:
@DeboDevelop - Associate the intent with a chatbot as well.

Reply:
@AdityaTiwari2102 Done.